app/lydia/src/main.cpp

```diff
@@ -56,20 +56,21 @@ int main(int argc, char **argv) {
   app.add_flag("-n,--no-empty", no_empty, "Enforce non-empty semantics.");
 
   // define --ltlf and --ldlf
-  auto formula_group = app.add_option_group("formula");
-  std::string ldlf;
-  std::string ltlf;
-  formula_group->require_option();
-  CLI::Option *ldlf_opt =
-      formula_group->add_option("--ldlf", ldlf, "LDLf formula.");
-  CLI::Option *ltlf_opt =
-      formula_group->add_option("--ltlf", ltlf, "LTLf formula.");
-  // --ldlf and --ltlf are mutually exclusive.
-  ldlf_opt->excludes(ltlf_opt);
-  ltlf_opt->excludes(ldlf_opt);
-
-  bool file_flag = false;
-  app.add_flag("-f,--file", file_flag, "File flag.");
+  bool ldlf = false;
+  app.add_flag("--ldlf", ldlf, "LDLf formula.");
+  bool ltlf = false;
+  app.add_flag("--ltlf", ltlf, "LTLf formula.");
```

Comment:
shouldn't this be mutually exclusive? @francescofuggitti

Reply:
yes, but is it possible to specify such property for flags??